pkg/postage/service_test.go

```diff
@@ -28,7 +28,7 @@ func TestSaveLoad(t *testing.T) {
 			t.Fatal(err)
 		}
 		for i := 0; i < 16; i++ {
-			ps.Add(newTestStampIssuer(t, 1000))
+			_ = ps.Add(newTestStampIssuer(t, 1000))
```

Comment:
It would be better to add
```
if err != nil {
    t.Fatal(err)
}
```
If anything changes in the implementation later we need to catch that.

Same for all test calls changed.